working-groups/best-practices.md

```diff
@@ -2,23 +2,38 @@
   * Determine and document which types of decisions require proactive approval from the WG.  For most WGs, most work should be able to proceed without requiring an affirmative approval of the WG (e.g. code reviews and smaller feature changes) unless specifically requested by someone in the WG.
   * Document how members and non-members of the WG may request that an item be brought to the WG for a decision and how decisions will be documented.  One option is to add issues requiring WG decisions to a “Pending WG Decision” project.
-* Status updates.
+  
+## Status updates
   * Publish periodic status updates (ideally at least once every two weeks) so the community can keep up to date on the WG’s work without having to follow all issues/Slack conversations/etc.
-  * These updates should include:
-    * what the WG accomplished since the last status update
-    * what they are planning on working on before the next status update
-    * any lessons the WG learned they’d like to share with others in the AMP community
-  * File these updates as issues in the WG’s repo with the label “Type: Status Update.”
+    *  These updates should include:
+      * what the WG accomplished since the last status update
+      * what they are planning on working on before the next status update
+      * any lessons the WG learned they’d like to share with others in the AMP community
+    * File these updates as issues in the WG’s repo with the label “Type: Status Update.”
+  * Each WG will provide a higher-level update to the community/TSC approximately every three months at a Design Review.  See the [Working Group Updates page](https://github.com/ampproject/meta/wiki/Working-Group-Updates) for more information.
+  
+## Working Group Membership
+  * According to [AMP's governance model](https://github.com/ampproject/meta/blob/master/GOVERNANCE.md), the TSC sets the initial membership of each Working Group.  Each Working Group then adjusts its membership as needed using the [consensus-based decision-making policy](../GOVERNANCE.md#decision-making-policy).
+  * Although each Working Group may define its own standards for determining Working Group membership, the following is a set of baseline expectations for Working Group members that most Working Groups follow:
+    * Members of a Working Group are expected to be contributing in a significant way to the work of the Working Group.  Significant code contributions is one way to demonstrate this, but you can also demonstrate this through things like driving an overarching product vision for the WG.
+    * Working Group members should be actively participating in WG discussions and generally aware of/involved in the work of the WG beyond their own projects.  Members of a WG may bring their own perspective (and the perspective of their employer) into the WG, but their participation in the WG should not be limited to only projects they/their employer prioritize.
```

Comment:
Maybe this isn't the best place to bring this up, but what defines actively? Are we talking like daily standups? Or should every working group have like a monthly open chat about what they are trying to accomplish and things?

Reply:
I wanted to avoid being too specific on these bullet points both to avoid being too checkboxy ("if I attend N standups I should be a WG member" and "you can't be a member because you didn't attend N standups") and because the definition of what participating actively means will differ depending on the Working Group.

What I'm mainly trying to capture here is that to be a WG member you should be involved in work in the WG other than your own individual project, and you're participating in discussions/etc. at around the same level as other members of the WG.

Can you think of a better way to capture this?

Reply:
@mrjoro Ahh I see what you are saying now. Cool, that makes a lot of sense 😄